examples/tts/speechllm/conf/megatron_t5_speechlm_model.yaml

```diff
@@ -71,7 +69,7 @@ model:
   context_conditioning: decoder
   train_from_scratch: true
   override_tokenizer_vocab_file: ???
-  use_flash_attention: true
+  use_flash_attention: false
```

Comment:
I have trained with flash_attention. TBH, I haven't seen any performance speed impact nor any training / val loss impact. Since we add the attention prior, we only use flash attention in certain attention layers (self-attention in encoder, self-attention in decoder, cross-attention in decoder ONLY when we turn off the prior, don't use ctc loss, or don't want to return the attention probabilities). Cross-attention in decoder only uses flash attention during validation on iterations where we don't plot attention probabilities.

Reply:
i see. Previous recipes default as True, so from now on, we all start with False.